app/assets/src/components/views/amr_heatmap/AMRHeatmapView.jsx

```diff
@@ -185,10 +206,38 @@ export default class AMRHeatmapView extends React.Component {
     }
   }
 
+  getDownloadCSVLink(csvOption) {
+    const dataString = this.computeHeatmapValuesForCSV();
+    const dataBlob = new Blob([dataString], { type: "text/csv" });
+    return (
+      <a
+        href={URL.createObjectURL(dataBlob)}
+        download="idseq_amr_heatmap_values.csv"
+        target="_blank"
+        rel="noopener noreferrer"
+        key={`${csvOption.text}_csv_link`}
+      >
+        {csvOption.text}
+      </a>
+    );
+  }
+
+  getDownloadOptions() {
+    const downloadOptions = DOWNLOAD_OPTIONS;
```

Comment:
I think you can make this function cleaner by just doing:
```
return {
  value: "csv",
  text: this.getDownloadCSVLink()
}
```

Don't think you need the additional variables.